server/models/Tier.js

```diff
@@ -319,6 +319,10 @@ export default function(Sequelize, DataTypes) {
 
   // TODO: Check for maxQuantityPerUser
   Tier.prototype.availableQuantity = function() {
```

Comment:
Can we somehow consider dropping this method?

Reply:
It's used by `checkAvailableQuantity` below. We can eventually merge the two functions, but I think being able to do `tier.availableQuantity()` just to know how many are left is also nice.